### PR TITLE
SUMO-210897 - Release note for deactivation of unused access keys

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a release note to the [Service release notes](https://help.sumologic.com/release-notes-service/) for the release of the new feature to automatically deactivate access keys.

This is a companion PR to #1985.

Issue number: 
[SUMO-210897](https://jira.kumoroku.com/jira/browse/SUMO-210897)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-service/2023-04-14-deactivation-of-unused-access-keys.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - access keys
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+By default API access keys will now automatically be deactivated if they have been unused for 30 days. This enhances the security of your account by reducing the risk of unauthorized access through inactive access keys. This ensures that only active access keys can be used to access your account and its resources.
```
Comment (stacykor) [inline blog-service/2023-04-14-deactivation-of-unused-access-keys.md:14]: Usually we preface the release note with "we'd like to announce" for releases and for behavior changes "We'd like to update you on the following change"

**Review by stacykor — CHANGES_REQUESTED** [review]

File: blog-service/2023-04-14-deactivation-of-unused-access-keys.md (hunk 1 above)
Comment (jpipkin1) [inline blog-service/2023-04-14-deactivation-of-unused-access-keys.md:14]:
```suggestion
We'd like to announce that by default API access keys will now automatically be deactivated if they have been unused for 30 days. This enhances the security of your account by reducing the risk of unauthorized access through inactive access keys. This ensures that only active access keys can be used to access your account and its resources. 
```

**Review by kimsauce — APPROVED** [review]

File: blog-service/2023-04-14-deactivation-of-unused-access-keys.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - access keys
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
```
Comment (kimsauce) [inline blog-service/2023-04-14-deactivation-of-unused-access-keys.md:12]: Since there's no image, you don't need this

```suggestion
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor fix

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce. I made your requested change.

**Review by kimsauce — APPROVED** [review]

**Review by stacykor — APPROVED** [review]
LGTM

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
Hi John! I saw this release note was date today. Will be moving out?

**Comment by jpipkin1** [comment]
Thanks for your vigilance, @kimsauce! I got word that the feature will go out on May 29. I updated [SUMO-210897](https://jira.kumoroku.com/jira/browse/SUMO-210897) with the new date.

File: blog-service/2023-06-06-security.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+keywords:
+  - access keys
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+
+We're happy to announce that by default API access keys will now automatically be deactivated if they have been unused for 30 days. This enhances the security of your account by reducing the risk of unauthorized access through inactive access keys. This ensures that only active access keys can be used to access your account and its resources.
```
Comment (kimsauce) [inline blog-service/2023-06-06-security.md:13]:
```suggestion
We're happy to announce that by default, API access keys will now automatically be deactivated if they have been unused for 30 days. This enhances the security of your account by reducing the risk of unauthorized access through inactive access keys. This ensures that only active access keys can be used to access your account and its resources. 
```

**Comment by jpipkin1** [comment]
On June 1, updated the release note per comments from @kevin-sumo submitted via Slack.

**Pull request merged**